rust/core-lib/src/rpc.rs

```diff
@@ -175,9 +176,11 @@ impl<'a> CoreCommand<'a> {
                                 .map(|cmd| Edit { view_id: ViewIdentifier::from(view_id), edit_command: cmd })
                         } else { Err(MalformedCoreParams(method.to_string(), params.clone())) }
                 }),
-                "plugin" => serde_json::from_value::<PluginCommand>(params.clone())
-                    .map(|cmd| Plugin { plugin_command: cmd })
-                    .map_err(|_| MalformedPluginParams(method.to_string(), params.clone())),
+            "plugin" => serde_json::from_value::<PluginCommand>(params.clone())
+                .map(|cmd| Plugin { plugin_command: cmd })
+                .map_err(|_| MalformedPluginParams(method.to_string(), params.clone())),
+
+            "client_started" => Ok(ClientStarted{}),
```

Comment:
I'm mildly surprised to see that this compiles (including the braces `{}` in a a non-struct enum member). I'm tempted to say remove them, for clarity, but it's a bit of a nit.